generic_examples.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+make clean && make
+
+make run &
+
+# wait until program is ready to listen
+# not sure how else to it now
+sleep 3;
+
+echo "<TEST DATA SEND START>"
+# valid
+echo "VALID CASES:"
+./test/data/generic/valid.sh
+# invalid
+echo "INVALID CASES:"
+./test/data/generic/invalid.sh
+echo "<TEST DATA SEND END>"
+
+> debug
+pid=$(pgrep pcp-statsd)
+kill -USR1 $pid
+# I have no idea how to watch output of parallel task and block until given text is found
+sleep 5
+kill -TERM $pid
```

Comment:
Just an idea - these example shell scripts could all live in `test/` directory. They are kindof tests. But it's fine here.